storage/gcloud/aio/storage/storage.py

```diff
@@ -540,8 +544,13 @@ async def _initiate_upload(self, url: str, object_name: str,
         params['uploadType'] = 'resumable'
 
         metadict = (metadata or {}).copy()
-        metadict.update({'name': object_name})
-        metadata_ = json.dumps(metadict)
+        custom_metadata = metadict.pop('metadata', None)
```

Comment:
Same question, looks like this block could be copied from the above:
```python
metadict= {self._format_metadata_key(k): v
            for k, v in metadata.items()}
metadict['name'] = object_name
metadata_ = json.dumps(metadict)
```

Reply:
Absolutely... I'll correct that.